src/app/dataset-block/metadata-block/components/event-details/components/common/simple-property/simple-property.component.html

```diff
@@ -0,0 +1 @@
+<span [class]="class">{{ data }}</span>
```

Comment:
`class` is optional in the component. Here it is used without checks. If we always have a class, change the the property or have a default value maybe.